module/Paprika/src/Model/Product/MAINBOARD.php

```diff
@@ -0,0 +1,264 @@
+            'mainboard_NumberPCIeSlotsx16x16'                => $this->NumberPCIeSlotsx16x16,
+            'mainboard_NumberPCIeSlotsx16x8'                => $this->NumberPCIeSlotsx16x8,
+            'mainboard_NumberPCIeSlotsx16x4'                => $this->NumberPCIeSlotsx16x4,
+            'mainboard_NumberPCIeSlotsx8'                => $this->NumberPCIeSlotsx8,
+            'mainboard_NumberPCIeSlotsx4'                => $this->NumberPCIeSlotsx4,
+            'mainboard_NumberPCIeSlotsx1'                => $this->NumberPCIeSlotsx1,
+            'mainboard_NumbermSata'                => $this->NumbermSata,
+            'mainboard_NumberM22280Slots'                => $this->NumberM22280Slots,
+            'mainboard_NumberM222110Slots'                => $this->NumberM222110Slots,
+            'mainboard_NumberSata6GBs'                => $this->NumberSata6GBs,
+            'mainboard_NumberSata3GBs'                => $this->NumberSata3GBs,
+            'mainboard_Soundcodec'                => $this->Soundcodec,
+            'mainboard_TypeAudio'                => $this->TypeAudio,
+            'mainboard_InterfaceAudio'                => $this->InterfaceAudio,
+            'mainboard_InterfaceLan'                => $this->InterfaceLan,
+            'mainboard_TypeBluetooth'                => $this->TypeBluetooth,
+            'mainboard_TypeWifi'                => $this->TypeWifi,
+            'mainboard_NumberUSB20'                => $this->NumberUSB20,
+            'mainboard_NumberUSB30'                => $this->NumberUSB30,
+            'mainboard_NumberUSB31'                => $this->NumberUSB31,
+            'mainboard_NumberUSBC'                => $this->NumberUSBC,
+            'mainboard_NumberPS2'                => $this->NumberPS2,
+            'mainboard_TypeBios'                => $this->TypeBios,
+            'mainboard_StylePackage'                => $this->StylePackage,
+            'mainboard_misc'                  => $this->Misc,
+            'mainboard_price'                 => $this->Price,
+        ];
+    }
+    /**
+     * Set input filter
+     *
+     * @param InputFilterInterface $inputFilter
+     */
+    public function setInputFilter(InputFilterInterface $inputFilter)
+    {
+        throw new DomainException(sprintf(
+            '%s does not allow injection of an alternate input filter',
+            __CLASS__
+        ));
+    }
+    /**
+     * Get input filter
+     *
+     * @return InputFilter
+     */
+    public function getInputFilter()
+    {
+        if ($this->inputFilter) {
+            return $this->inputFilter;
+        }
+        $inputFilter = new InputFilter();
+        $inputFilter->add([
+            'name' => 'mainboard_id',
+            'required' => true,
+            'filters' => [
+                ['name' => ToInt::class],
+            ],
+        ]);
+        $inputFilter->add([
+            'name' => 'mainboard_model_name',

```

Comment:
Array align

Reply:
Meinst du ich soll die Elemente anders einrücken? also z.B. bei Zeile 244
```suggestion
            'name' => 'mainboard_model_name',
```

Reply:
```
$inputFilter->add(
    [
        'name'      => 'mainboard_id',
        'required' => true,
            'filters'  => [
                ['name' => ToInt::class],
            ],
        ]
);
```
So bitte. strg+alt+l ist in einige IDEs der shortcut das es automatisch die syntax aufräumt